app/src/message_handler.c

```diff
@@ -205,6 +207,11 @@ bool ltv_found(struct bt_data *data, void *user_data)
 		memcpy(&_parsed->broadcast_code, &data->data[0], BT_AUDIO_BROADCAST_CODE_SIZE);
 		LOG_HEXDUMP_DBG(_parsed->broadcast_code, BT_AUDIO_BROADCAST_CODE_SIZE, "broadcast code:");
 		return true;
+	case BT_DATA_BIS_SYNC:
+		_parsed->bis_sync[_parsed->num_subgroups] = sys_get_le32(data->data);
+		LOG_DBG("bis_sync[%u]: 0x%08x", _parsed->num_subgroups, _parsed->bis_sync[_parsed->num_subgroups]);
+		_parsed->num_subgroups++;
+		return true;
```

Comment:
Could we have this in one BT_DATA_BIS_SYNC field? - I am not sure ordering can be guaranteed otherwise (but I will take a look)

Reply:
Yes. But then we should have number of subgroups included in BT_DATA_BIS_SYNC data